astropy/wcs/wcsapi/fitswcs.py

```diff
@@ -395,6 +395,8 @@ def _get_components_and_classes(self):
                             location = EarthLocation(*self.wcs.obsgeo[:3], unit=u.m)
                     elif trefpos == 'geocenter':
                         location = EarthLocation(0, 0, 0, unit=u.m)
+                    elif trefpos == '':
```

Comment:
I guess it is a real API change outside of testing, so I'll wait for @Cadair or @nden to have a look first.

Reply:
This is above my pay grade :stuck_out_tongue:

Reply:
It's an affects-dev change, just fixes a bug that was introduced in a recent PR.